tests/sys/psa_crypto_hashes/sha3.c

```diff
@@ -0,0 +1,44 @@
+#include "psa/crypto.h"
+
+static const uint8_t msg[] = "Hello World!";
+static const size_t msg_len = sizeof(msg)-1; // exclude NULL-byte
+
+static const uint8_t hash_sha3_256[] ={
+    0xd0, 0xe4, 0x74, 0x86, 0xbb, 0xf4, 0xc1, 0x6a, 0xca, 0xc2, 0x6f, 0x8b, 0x65, 0x35, 0x92, 0x97, 0x3c, 0x13, 0x62, 0x90, 0x9f, 0x90, 0x26, 0x28, 0x77, 0x08, 0x9f, 0x9c, 0x8a, 0x45, 0x36, 0xaf
```

Comment:
Please keep the line length below 80 chars, see the RIOT [coding convention](https://github.com/RIOT-OS/RIOT/blob/master/CODING_CONVENTIONS.md) (same below). The CI will complain in any case :)